src/main/webapp/app/scores/exercise-scores.component.html

```diff
@@ -206,6 +206,7 @@ <h2>
                         *ngIf="newManualResultAllowed"
                         [participationId]="value.participation.id"
                         [latestResult]="value"
+                        (onResultCreated)="refresh()"
```

Comment:
I think this change is not a good idea. In exercises with lots of participants, this would mean that every time a tutor creates or edits a manual result, all results need to be reloaded and the whole page needs to be re-rendered. I think there must be a simpler solution.

I also have noticed that we currently only update the result string and not the entire row here, but could we not use the passed new / changed result in this event and only update this single instance in the results array of the component?

I will merge this for now, but I think we should have a follow up PR where we improve this!